src/frequenz/dispatch/_dispatcher.py

```diff
@@ -200,7 +203,10 @@ def new_lifecycle_events_receiver(
         return self._bg_service.new_lifecycle_events_receiver(dispatch_type)
 
     async def new_running_state_event_receiver(
-        self, dispatch_type: str, *, unify_running_intervals: bool = True
+        self,
+        dispatch_type: str,
+        *,
+        merge_strategy: MergeStrategy = NoMerge(),
```

Comment:
OK, I see you have a default in here, but again I would make it `| None = None`.